public/js/services/AtomsApi.js

```diff
@@ -21,7 +21,8 @@ export default {
         credentials: 'same-origin',
         body: JSON.stringify(atomInfo),
         headers: {
-          'Content-Type': 'application/json'
+          'Content-Type': 'application/json',
+          'Csrf-Token': window.guardian.csrf.token
```

Comment:
These would throw if window.guardian is undefined -- though possibly not an issue since csrf will always be defined?

Reply:
Indeed. I got half way through adding the multiple layers of `undefined` checks, but decided that it will always be defined, and if it isn't defined the http call would fail anyway.